content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
-=============================
-Assign leads based on scoring
-=============================
+=========================================
+Assign leads with predictive lead scoring
+=========================================
 
-*Lead Scoring* lets you rank and prioritize leads based on selected criteria. A higher score
-indicates that a lead is more likely to bring value to your company.
+The *Odoo CRM* app can automatically assign leads/opportunities to sales teams and salespeople. A
+standard practice is to assign leads based on the probability of winning each lead. Companies can
+prioritize the leads that are more likely to result in successful deals by quickly assigning them
+to the appropriate salespeople.
```

Comment:
The Odoo *CRM*...

I believe is how ZST would this formatted. So, be sure to go back and adjust those accordingly.